src/Surfnet/ServiceProviderDashboard/Domain/ValueObject/Contact.php

```diff
@@ -45,6 +46,19 @@ class Contact
      */
     private $phone;
 
+    public static function from(?ContactEntity $contact)
```

Comment:
Please always return a concrete implementation of a Contact or consider using a null object.